src/utils.ts

```diff
@@ -0,0 +1 @@
+export const backendURL:string='http://localhost:3000/'
```

Comment:
**suggestion:** you can use `environment.ts` file to keep this or you can have a proxy setup for the project.